index.html

```diff
@@ -162,7 +162,7 @@ <h1 class="heading"><i class="fas fa-cloud"></i> sButton CDN</h1>
             When you only need to include SButtons’s compiled CSS , you can use
             SButtonCDN.
             <br /><br />
-            <a href="/examples.html" class="sbtn transparent-btn banner-link"
+            <a href="/documentation.html" class="sbtn transparent-btn banner-link"
```

Comment:
Please remove the backslash in the beginning

Reply:
Just did it.